.github/workflows/rust.yml

```diff
@@ -247,3 +247,13 @@ jobs:
       with:
         name: cargo-doc
         path: target/doc
+  deploy:
+    needs: bench
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build docker image
+        run: docker build ./
+      - name: Docker login
+        run: docker login -u humb1t -p $DOCKERHUB 
```

Comment:
Should we publish with company / hyperledger accounts?

Reply:
I was added as an admin to hyperledger. There is no Soramitsu account with such rights.

Reply:
But bench is actually not the last one:
- check -> test -> bench
- check -> test-docker
They will be running in parallel and we don't have a guarantee that bench completes the last and the test-docker may fail while bench succeeds.